test/integration/pj-rehearse.sh

```diff
@@ -43,6 +43,9 @@ export JOB_SPEC='{"type":"presubmit","job":"pull-ci-openshift-release-master-reh
 actual="${workdir}/rehearsals.yaml"
 os::cmd::expect_success "pj-rehearse --dry-run=true --candidate-path ${repo} --rehearsal-limit 20 > ${actual}"
 os::integration::sanitize_prowjob_yaml ${actual}
+# Substitute the SHA in the job names to a known SHA for comparison
+sed -i -E -e "s/-${candidate_sha}-/-4de8ab7c20656998264a2593116288f5eb070b32-/g" ${actual}
```

Comment:
This will break the manifest since the limit is to 63 chars

Reply:
I assume you mean using the SHA will break it and not the substitution for the integration tests. In that case, do you have any idea of what else we can use absent the `BuildID`?

Reply:
Yea I mean that this will generate https://github.com/openshift/ci-tools/pull/3063/files#diff-e0b6c8feb0bd8af9facc04c0a2e3384bf1258e99ea6c5404cdf25abb998af9bfR132 which is not allowed in Kubernetes. We could use something else for the tests though. Maybe a 5digit hash code with the job itself as an input?

Reply:
Is it important that it is unique (and deterministic) to a specific HEAD of the PR? I thought that was why the `BuildID` was being used in the first place, but I could have made a poor assumption. If that _is_ the case, maybe we can do a hash code based on the SHA...

Reply:
For the record: The limit is 253 chars for the object names, and 64 chars is for the label values.